src/fides/api/service/saas_request/saas_request_override_factory.py

```diff
@@ -42,8 +44,8 @@ class SaaSRequestOverrideFactory:
 
     @classmethod
     def register(cls, name: str, request_types: List[SaaSRequestType]) -> Callable[
-        [Callable[..., Union[List[Row], int, None]]],
-        Callable[..., Union[List[Row], int, None]],
+        [Callable[..., Union[List[Row], int, bool, None]]],
+        Callable[..., Union[List[Row], int, bool, None]],
```

Comment:
may be time for us to alias this type annotation 😛 

don't need to adjust here, but throwing it out here in case you want to!